kong/clustering/config_helper.lua

```diff
@@ -0,0 +1,230 @@
+local constants = require("kong.constants")
```

Comment:
It seems this file is only doing the hash function. Consider renaming to `kong/clustering/config_hash.lua` and making it return the function directly.

Reply:
`utils.lua` already has too many functions, `config_helper.lua` contains all code about config, including hash and update, I think we can keep it.